src/layer/poi.js

```diff
@@ -0,0 +1,74 @@
+import * as label from "../constants/label.js";
+import * as Color from "../constants/color.js";
+
+export const poi = {
+  id: "poi",
```

Comment:
In legend_config.js, add a new “Points of interest” section to `sections` that contains an entry for each of the distinct icons rendered by this layer. Each entry’s filter would likely be based on the `class` feature property.